cashews/decorators/rate.py

```diff
@@ -13,7 +13,7 @@ class RateLimitException(Exception):
     pass
 
 
-def _default_action(*args, **kwargs):
+def _default_action(*args: Any, **kwargs: Any) -> RateLimitException:
```

Comment:
I am not sure that is right that it return RateLimitException - it is raised it. Is it the same?

Reply:
I overlooked that we have `raise` here instead of `return`. My type hint here is wrong. I've fixed.